ios/Runner/AppDelegate.swift

```diff
@@ -99,6 +99,17 @@ import Starscream
             }
         })
         
+        let localeChannel = FlutterMethodChannel(name: "locale_measurement",
```

Comment:
We might use this channel to get other settings. I suggest using the name `locale` only.